SUBSCRIBE_HISTORY_FORMATTING_PLAN.md

```
@@ -0,0 +1,389 @@
+# Implementation Plan: Consistent Message Output for channels subscribe & history
+
+## Original Request
+
+Currently when running `bin/run.js channels subscribe test`, the output is:
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)